jit/v2/JIT_v2.0.0.yaml

```diff
@@ -940,6 +936,263 @@ components:
         example: '2'
       description: |
         An API-Version header **MAY** be added to the request (optional); if added it **MUST** only contain **MAJOR** version. API-Version header **MUST** be aligned with the URI version.
+    ##############
+    # Query params
+    ##############
+
+    # Technical query parameters
+
+    portCallIDQueryParam:
+      in: query
+      name: portCallID
+      description: |
+        The **Port Call Identifier** to filter by. Specifying this filter will only return **Port Call Service** objects including the `portCallID`.
+      schema:
+        type: string
+        format: uuid
+        example: 0342254a-5927-4856-b9c9-aa12e7c00563
+    transportCallIDQueryParam:
+      in: query
+      name: transportCallID
+      description: |
+        The **Transport Call Identifier** to filter by. Specifying this filter will only return **Port Call Service** objects including the `transportCallID`.
+      schema:
+        type: string
+        format: uuid
+        example: 0342254a-5927-4856-b9c9-aa12e7c00563
+    portCallServiceIDQueryParam:
+      in: query
+      name: portCallServiceID
+      description: |
+        The **Port Call Service Identifier** to filter by. Specifying this filter will (at most) return a single **Port Call Service** objects including the `portCallServiceID` (if the `portCallServiceID` exists).
+      schema:
+        type: string
+        format: uuid
+        example: 0342254a-5927-4856-b9c9-aa12e7c00563
+
+    # Timestamp query parameters
+
+    startDateTimeQueryParam:
+      in: query
+      name: startDateTime
+      description: |
+        The **start date time** of the period for which **Port Call Service** information is requested. If a date and time of any **Timestamp** inside a **Port Call Service** matches a date and time on or after (≥) the `startDateTime` **Port Call Service** will be included in the result. All matching is done towards local Date and time at the place of the**Port Call Service**. The matched **Timestamp** in the timestamps of the **Port Call Service** must be included in the **Port Call Service** objects returned.
```

Comment:
after ---> or after (≥) the `startDateTime`, shouldn't there be a 'that' 'or those' ? Is it just my feeling that a connection is missing there?

A missing space -place of the**Port Call Service**.

Reply:
Tried to improve the text - let me know if you think it is better...
Fixed the missing space